src/clr-angular/emphasis/alert/providers/multi-alert.service.ts

```diff
@@ -50,6 +50,11 @@ export class MultiAlertService {
 
   manage(alerts: QueryList<ClrAlert>) {
     this.allAlerts = alerts;
+    this.allAlerts.changes.subscribe(() => {
```

Comment:
Do you need to unsubscribe to an observable that is on the same object? I suspect so?